.github/workflows/pull_request.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 16
```

Comment:
docker file is set to 16 please update it to ts-server boiler plate version (20),  same as with the workflow file